src/triage/component/architect/feature_generators.py

```diff
@@ -351,6 +352,46 @@ def generate_all_table_tasks(self, aggregations, task_type):
         logging.info('Created %s tables', len(table_tasks.keys()))
         return table_tasks
 
+    def create_preimputed_features(self, feature_aggregation_config, feature_dates):
+        """Create only preimputed features for a set of dates"""
+        all_tasks = self.generate_all_table_tasks(
+            self.aggregations(
+                feature_aggregation_config,
+                feature_dates,
+                state_table=None
+            ),
+            task_type='aggregation'
+        )
+        logging.info('Generated a total of %s table tasks', len(all_tasks))
+        for i, task in enumerate(all_tasks.values()):
```

Comment:
FYI you can tell `enumerate` to start from `1`:

    for (task_num, task) in enumerate(all_tasks.values(), 1):
        …